lib/services/local/access_list_test.go

```diff
@@ -135,7 +135,7 @@ func TestAccessListCRUD(t *testing.T) {
 	require.True(t, trace.IsAlreadyExists(err))
 }
 
-func TestAccessListDedupeOwnersBackwardsCompat(t *testing.T) {
+func TestAccessListCreateLimit_UpsertAccessList(t *testing.T) {
```

Comment:
This is a long test. Can you add a comment and explain the scenario that we're testing here?